src/generator.py

```diff
@@ -1,6 +1,11 @@
 from maze import Maze
+import sys
 
-maze = Maze(20)
-maze.generate_maze()
-maze.print()
-print("Welcome to 2D maze")
+try:
+    maze_size = int(sys.argv[1])
+    maze = Maze(maze_size)
+    maze.generate_maze()
+    maze.print()
+    print("Welcome to 2D maze")
+except:
+    print("Exception Occurred. Maze size not given or incorrect format.")
```

Comment:
Good exception description.

Reply:
Thanks. I wasn't sure about adding a desired format (e.g. "Expected Format: <file> 'size' ") but I might add that in the future.